esp/src/src-react/components/forms/landing-zone/FileListForm.tsx

```diff
@@ -63,7 +63,7 @@ export const FileListForm: React.FunctionComponent<FileListFormProps> = ({
                 formData.append("uploadedfiles[]", file);
             });
             const uploadUrl = "/FileSpray/UploadFile.json?" +
-                "upload_&rawxml_=1&NetAddress=" + machine + "&OS=" + os + "&Path=" + data.path;
+                "upload_&rawxml_=1&NetAddress=" + machine + "&OS=" + os + "&Path=" + data.path + pathSep;
```

Comment:
There should probably be a check on `data.path` in case it already ends with a `pathSep`?